dev/test/index.ts

```diff
@@ -883,7 +879,8 @@ describe('getAll() method', () => {
     const overrides = {batchGetDocuments: () => stream(found('documentId'))};
 
     return createInstance(overrides).then(firestore => {
-      return firestore.getAll(firestore.doc('collectionId/documentId'))
+      return (firestore as InvalidApiUsage)
+          .getAll([firestore.doc('collectionId/documentId')])
```

Comment:
If this is expected to work, we shouldn't have to cast to `InvalidApiUsage`, right? That would imply that we might be breaking someone's typescript project?

Reply:
This was never supported for TypeScript users, only for JavaScript users.

This change does however break the no-argument call to `getAll()`. Before, it was allowed to call `getAll()` with no references, now one needs to be provided. This is a compile-time error and we are bumping the minor version.

We should consider removing the array support when we go GA: https://b.corp.google.com/issues/119353405